gcs-action/__tests__/gcs.test.ts

```diff
@@ -0,0 +1,60 @@
+import * as gcs from '../src/gcs';
+import { UploadResponse } from '@google-cloud/storage';
+
+/*
+ * Test upload a single file to GCS
+ */
+test('upload file to gcs without prefix', async () => {
```

Comment:
These seem more like ITs than unit tests, as they'll need existing infra and an SA to run. Let's segregate them into a separate *.int.test.ts file. Ideally unit tests should be testing the individual behaviors of the supporting logic without needing to hit live resources.

Reply:
Done